package.json

```diff
@@ -119,6 +120,7 @@
     "enzyme-shallow-equal": "^1.0.0",
     "is-touch-device": "^1.0.1",
     "lodash": "^4.1.1",
+    "luxon": "^1.22.0",
```

Comment:
this needs to be a peer dep, not a regular dep, because it's critical that the user and the library be using the same version of luxon.

Reply:
Thanks.  Probably did `yarn install` while testing.